README.md

```diff
@@ -57,6 +67,10 @@ Modify your project's `package.json` so that the `jest` section looks something
 ```
 This setup should allow you to write Jest tests in Typescript and be able to locate errors without any additional gymnastics.
 
```

Comment:
We should also add the installation instructions for yarn users

```
yarn add ts-jest --dev
```

What do you think?